ui/imports/shared/status/StatusEmojiPopup.qml

```diff
@@ -65,28 +67,25 @@ StatusDropdown {
     QtObject {
         id: d
 
-        readonly property string searchStringLowercase: searchBox.text.toLowerCase()
+        readonly property string searchString: searchBox.text
         readonly property int headerMargin: 8
-        readonly property int imageWidth: 26
-        readonly property int imageMargin: 4
+        readonly property int imageWidth: 32
+        readonly property int imageMargin: 6
 
         readonly property var filteredModel: SortFilterProxyModel {
             sourceModel: root.emojiModel
 
             filters: [
-                FastExpressionFilter {
-                    expression: {
-                        if (model.category === root.emojiModel.recentCategoryName) // don't show/duplicate recents when searching
-                            return false
-
-                        d.searchStringLowercase
-                        return model.name.toLowerCase().includes(d.searchStringLowercase) ||
-                                model.shortname.toLowerCase().includes(d.searchStringLowercase) ||
-                                model.aliases.some(a => a.includes(d.searchStringLowercase)) ||
-                                model.keywords.some(k => k.includes(d.searchStringLowercase))
```

Comment:
Omitting the aliases and keywords on purpose; they just contain tokenized `name` or `shortname`